EXPhotoViewer/EXPhotoViewer.m

```diff
@@ -58,6 +58,7 @@ -(void)loadView {
     UIImageView* imageView = [[UIImageView alloc] initWithFrame:self.view.bounds];
     imageView.contentMode = UIViewContentModeScaleAspectFill;
```

Comment:
actually not using cocoapod - so fine by me

Good luck merging it up - it seems forgotten (and it's a pity because
that's whats showing up on google search)

On Wed, Jan 28, 2015 at 10:31 PM, Spencer Comerford <
notifications@github.com> wrote:

> In EXPhotoViewer/EXPhotoViewer.m
> <https://github.com/ShowMobile/EXPhotoViewer/pull/1#discussion_r23720741>:
>
> > @@ -58,6 +58,7 @@ -(void)loadView {
> >      UIImageView* imageView = [[UIImageView alloc] initWithFrame:self.view.bounds];
> >      imageView.contentMode = UIViewContentModeScaleAspectFill;
>
> No prob, and thank you! Are you using cocoapods? I'll update the pod spec
> later but we'll either have to change the name of the pod or get the
> original repo owner to merge in the changes. Until then you'll need a line
> like:
>
> pod 'EXPhotoViewer',        :git => 'https://github.com/ShowMobile/EXPhotoViewer.git', :commit => '807e807441fea0bc24c957b5a3e3c171c46e82ce'
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ShowMobile/EXPhotoViewer/pull/1/files#r23720741>.
>